tests/regression/runner.py

```diff
@@ -277,11 +282,11 @@ def test_send_transfer(self):
                           response[i]["content"] + ", status code = " +
                           response[i]["status_code"])
 
-        pass_case = [0, 1, 2, 3, 5, 6, 7, 8, 9, 10]
+        pass_case = [0, 1, 2, 5, 6, 7, 8, 9, 10]
```

Comment:
Why?

Reply:
unicode message is banned from #259 
https://github.com/DLTcollab/tangle-accelerator/pull/259/files#diff-bd972cd9032d23d6985f194c5a55be82R405
So test case No3 can't will return error in TA

Reply:
Append some note about Unicode encoding inside source code and/or git commit messages.

Reply:
Resolved yet?

Reply:
I have added comment in commit message